tests/foreman/api/test_host.py

```diff
@@ -383,6 +437,95 @@ def test_positive_create_with_compute_profile(self):
         self.assertEqual(host.compute_profile.read().name, profile.name)
 
     @run_only_on('sat')
+    @tier2
+    def test_positive_create_with_content_view(self):
```

Comment:
Basically, covered in https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentview.py#L94
But probably, that one is more basic